components/automl/create_model_for_tables/component.py

```diff
@@ -0,0 +1,44 @@
+from typing import NamedTuple
+
+
+def automl_create_model_for_tables(
+    gcp_project_id: str,
+    gcp_region: str,
+    display_name: str,
+    dataset_id: str,
+    target_column_path: str = None,
+    input_feature_column_paths: list = None,
+    optimization_objective: str = 'MAXIMIZE_AU_PRC',
+    train_budget_milli_node_hours: int = 1000,
+) -> NamedTuple('Outputs', [('model_path', str), ('model_id', str)]):
+    import sys
+    import subprocess
+    subprocess.run([sys.executable, "-m", "pip", "install", "google-cloud-automl==0.4.0", "--quiet", "--no-warn-script-location"], env={"PIP_DISABLE_PIP_VERSION_CHECK": "1"}, check=True)
```

Comment:
PIP3 ?

Reply:
recommended  using ' as quote delimiter  to be consistent across.

Reply:
Done.

Reply:
> PIP3 ?

Not so fast. It's a module name here, not the program name =)